client/src/components/admin/AdminReservation.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { useGetReservationInfo } from "../../hooks/useGetReservationInfo";
+
+export const AdminReseservations = () => {
+  const {
+    allReservations,
+    markReservationAsProcessedHandler,
+    deleteReservationHandler,
+  } = useGetReservationInfo();
+
+  const handleProcessedClick = async (reservationId) => {
+    await markReservationAsProcessedHandler(reservationId);
+  };
+
+  const handleDeleteClick = async (reservationId) => {
+    const confirm = window.confirm(
+      "Da li ste sigurni da zelite da obrisete rezervaciju?",
+    );
+    if (confirm) {
+      await deleteReservationHandler(reservationId);
+    }
+  };
+
+  return (
+    <>
+      <h1>Rezervacije</h1>
+      <ul className="reservation-list">
+        {allReservations.map((reservation) => (
```

Comment:
Create a new component called ReservationItem.jsx and work with it. It makes your life easier when you have a huge map in parent component

Reply:
Done